jobs/base_model_validator_errors_test.py

```diff
@@ -162,3 +166,35 @@ def test_model_expired_error(self):
             % (model.id, days))
 
         self.assertEqual(error.message, msg)
+
+
+class ModelInvalidCommitStatusTests(ValidatorErrorTestBase):
+    def test_model_invalid_commit_status(self):
+        model1 = MockCommitLogEntryModel(
```

Comment:
Please use the real model and delete the mock.

Reply:
Done